discord/commands/options.py

```diff
@@ -86,8 +114,9 @@ async def hello(
 
     Attributes
     ----------
-    input_type: :class:`Any`
-        The type of input that is expected for this option.
+    input_type: Union[Type[:class:`str`], Type[:class:`bool`], Type[:class:`int`], Type[:class:`float`], Type[:class:`.abc.GuildChannel`], Type[:class:`Thread`], Type[:class:`Member`], Type[:class:`User`], Type[:class:`Attachment`], Type[:class:`Role`], Type[:class:`.abc.Mentionable`], :class:`SlashCommandOptionType`, :class:`Converter`, Type[:class:`Converter`], Type[:class:`Enum`], Type[:class:`DiscordEnum`]]
```

Comment:
```suggestion
    input_type: Union[:class:`str`, :class:`bool`, :class:`int`, :class:`float`, :class:`.abc.GuildChannel`, :class:`Thread`, :class:`Member`, :class:`User`, :class:`Attachment`, :class:`Role`, :class:`abc.Mentionable`, :class:`SlashCommandOptionType`, :class:`.ext.commands.Converter`, :class:`enum.Enum`, :class:`Enum`]
```
I don't think `Type[]` is necessary for docs

Reply:
- I was too fast

Reply:
Oh yeah, so then
```suggestion
    input_type: Union[Type[:class:`str`], Type[:class:`bool`], Type[:class:`int`], Type[:class:`float`], Type[:class:`.abc.GuildChannel`], Type[:class:`Thread`], Type[:class:`Member`], Type[:class:`User`], Type[:class:`Attachment`], Type[:class:`Role`], Type[:class:`.abc.Mentionable`], :class:`SlashCommandOptionType`, Type[:class:`.ext.commands.Converter`], Type[:class:`enums.Enum`], Type[:class:`Enum`]]
```
should work

Reply:
yeah seems ok